src/watcher/chokidar.ts

```diff
@@ -36,25 +40,43 @@ export function watch(
   const watcher = chokidar.watch(paths, options) as FileWatcher
   const programmaticallyWatchedFiles: string[] = []
 
+  const isFileIgnored = (event: string, file: string) => {
```

Comment:
Hm, wonder if we can minimize the boolean flipping (find that hard to keep track of aka. boolean blindness).

Also would specialize this function to assume it is being run with `isSilentEvent` check pre-determined.

Helps to avoid overloading the term `ignore` here at all costs 😂🤯.

So in sum just a tiny helper then:

```suggestion
  const wasFileAddedSilently = (file: string) => {
```

Reply:
Like it a lot 👍